databricks_cli/jobs/cli.py

```diff
@@ -134,11 +139,16 @@ def get_cli(job_id):
 
 
 @click.command(context_settings=CONTEXT_SETTINGS)
-@click.option('--job-id', required=True, type=int)
-@click.option('--jar-params', default=None)
-@click.option('--notebook-params', default=None)
-@click.option('--python-params', default=None)
-@click.option('--spark-submit-params', default=None)
+@click.option('--job-id', required=True, type=JobIdClickType())
+@click.option('--jar-params', default=None, type=JsonClickType(),
+              help='JSON array of parameters. i.e. ["param1", "param2"]')
+@click.option('--notebook-params', default=None, type=JsonClickType(),
+              help='JSON map of key-value pairs. i.e. {"name": "john doe", "age": 35}')
+@click.option('--python-params', default=None, type=JsonClickType(),
```

Comment:
Do we support this? It is not in the API doc.

Reply:
It is supported and we'll add it to the API doc this release.